src/Components/Timer.js

```diff
@@ -46,11 +46,11 @@ const Timer = ({ isPolling, coffeeon }) => {
   };
 
   useEffect(() => {
-    if (isPolling == 'true' && !coffeeon) {
+    if (isPolling === 'true' && !coffeeon) {
```

Comment:
As we progress, move isPolling into rooms settings.

Coffeeon feature should be part of cards.